### DISCUSSION TITLE
pytest-bdd multi-languages support

### DISCUSSION THREAD

**Discussion by thufish** [pytest-dev/pytest-bdd#606] [Q&A]
Would pytest-bdd support multi-languages，such as chinese ?

**Comment by 552468054** [comment]
支持，只是关键词不支持，希望官方出8.0.0能支付关键词中文，我看behave都能关键词用中文编写

**Comment by youtux** [comment]
We already have an open issue for this, I simply don't have enough time to get to it: https://github.com/pytest-dev/pytest-bdd/issues/397

**Comment by jsa34** [comment]
This has now been implemented and release in version 8.0.0
